src/pivot-table/hooks/use-column-width/utils.ts

```diff
@@ -0,0 +1,30 @@
+import type { ColumnWidth } from "@qlik/nebula-table-utils/lib/components/ColumnAdjuster";
```

Comment:
I learned recently that `utils` is usually used as a name for code that is more of a tool, like a debugging tool or something, not just extracted logic. Maybe this should be called something different? helpers?

Reply:
Never heard that before. But since we use `utils` I think we should continue with that for now. If we like to it would probably be better to change "all" at once to something like "helper".

Reply:
I took a couse on codacademy, and they explained it in that way. Googling it again it is kind of vague, so I agree lets not change, especially for consistency